.github/workflows/solidity.yml

```diff
@@ -59,3 +59,9 @@ jobs:
       - name: protocol - Deploy L1 Contracts
         working-directory: ./packages/protocol
         run: pnpm deploy:hardhat
+
+      - name: relayer - Upload coverage to Codecov
```

Comment:
```suggestion
      - name: protocol - Upload coverage to Codecov
```

Reply:
Looks like this is still unresolved?

Reply:
ah, you can feel free to click "Commit suggestion" on these if youd like for things like this in the future : )

Done!